storage/innobase/include/fts0priv.inl

```diff
@@ -50,14 +50,14 @@ fts_write_object_id(
 	/* Use this to construct old(5.6.14 and 5.7.3) windows
 	ambiguous aux table names */
 	DBUG_EXECUTE_IF("innodb_test_wrong_windows_fts_aux_table_name",
-			return(sprintf(str, "%016llu", (ulonglong) id)););
+			return(snprintf(str, sizeof(str), "%016llu", (ulonglong) id)););
```

Comment:
The same thing - size of str is not known on compiling time, sizeof(str) will return the size of pointer not an array